www/src/css/snap.css

```diff
@@ -33,7 +33,10 @@
 
 .b-snap__actions .b-installer {
   margin:0 10px;
-  width:auto;
+}
+
+.b-snap__actions .b-installer__button {
```

Comment:
Instead of nested selectors it would be better to use modifiers on button, for example `.b-inteller__button_full-width`, but it's just a minor suggestion to consider in future.